### ISSUE TITLE
Blockquote (edit mode) - not accessible with screen readers

### ISSUE THREAD

**Issue by CarmenHilpert** [ckeditor/ckeditor5#19158]
## 📝 Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/docs/ckeditor5/latest/features/block-quote.html
2. Place cursor before the block quote
4. Press the arrow down key

### ✔️ Expected result

- The screen reader should announce the block quote when entering the section

### ❌ Actual result

- Screen reader output is as it is with normal text

## 📃 Other details

* Browser: Edge 138.0.3351.65
* OS: Windows 11 (10.0.22631 Build 22631)
* Screenrader: JAWS 2024
* First affected CKEditor version: not shure when it started - seen in v44.3 and later
* Installed CKEditor plugins: see demo page

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
